ssb-markdown.js

```diff
@@ -0,0 +1,33 @@
+const markdown = require('ssb-markdown')
```

Comment:
Mind combining these functions into the `ssb.js` file instead of having an extra one?

Reply:
That one needs to be an npm package. I've moved the functions from `./ssb-markdown.js` to `ssb.js` though, think that's what you meant :)